internal/contour/cache_test.go

```diff
@@ -361,38 +362,22 @@ func TestVirtualHostCacheValuesReturnsACopyOfItsInternalSlice(t *testing.T) {
 	if &v1[0] == &v2[0] {
 		// the address of the 0th element of the values slice should not be the same
 		// if it is, then we don't have a copy.
-		t.Fatalf("VirtualHostCache, consecutive calls to Values return the same backing slice: got: %p, want: %p", &v1[0], &v2[0])
-	}
-}
-
-func TestVirtualHostCacheValuesReturnsTheSameContents(t *testing.T) {
```

Comment:
Why did you delete this test?

Reply:
I had make a slight change in the implementation of `virtualHostCache` and I felt that this unit test is not relevant with the changes I made.